plugins/mission/mission.cpp

```diff
@@ -65,18 +65,20 @@ const char *Mission::result_str(Result result)
     switch (result) {
         case Result::SUCCESS:
             return "Success";
-        case Result::BUSY:
-            return "Busy";
         case Result::ERROR:
             return "Error";
         case Result::TOO_MANY_MISSION_ITEMS:
             return "Too many mission items";
-        case Result::INVALID_ARGUMENT:
-            return "Invalid argument";
+        case Result::BUSY:
```

Comment:
What's the link between `mission.cpp` and the `mission_raw` plugin?

Reply:
@JonasVautherin the mission plugin only returns values known to the SDK. The raw plugin returns all of them.

Reply:
Right, but I see a `Result` defined in `plugins/mission_raw`. This one is from `plugins/mission`, which is a different plugin. Do we need those changes in the `mission` plugin, too? And is that part of this PR?

Reply:
Oh right. I just found that one case was missing as part of copying from `mission`.